aws-lambda-java-events/src/main/java/com/amazonaws/services/lambda/runtime/events/APIGatewayV2WebSocketResponse.java

```diff
@@ -12,8 +14,8 @@
 
   private boolean isBase64Encoded = false;
   private int statusCode;
-  private Map<String, String> headers;
-  private Map<String, String[]> multiValueHeaders;
+  private HttpHeaders<String> headers;
+  private HttpHeaders<String[]> multiValueHeaders;
```

Comment:
If we're doing this change as part of v4, we should also change `String[]` to `List<String>` for consistency with other data models